src/viking.js

```diff
@@ -1,11 +1,79 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+
+};
+
+Soldier.prototype.attack = function () {
+    return this.strength;
+  };
+
+Soldier.prototype.receiveDamage = function(damage) {
+    this.health -= damage;
+  };
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+    this.name = name;
+
+    Soldier.call(this, health, strength);
+
+};
+
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+Viking.prototype.receiveDamage = function(damage) {
+    this.health -= damage;
+    if (this.health > 0) {
+        return this.name + " has received " + damage + " points of damage";
```

Comment:
You could try to use the template literals `${this.name} has received ...`  with `` to create the return string.